### PR TITLE
Point Of Sales v1.0

### PR DESCRIPTION
**Pull request by ric96**
Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,75 @@
...
+- [2) Software](#2-software)
+   - [2.1) Client Build Environment Setup](#21-client-build-environment-setup)
+   - [2.2) Database Server Setup](#22-api-keys)
+   - [2.3) Running the Application](#23-running-the-application)
+
+# 1) Hardware
+
+## 1.1) Hardware Requirements
+
+- [Any 96Boards CE](https://www.96boards.org/products/ce/) x3
```
Comment (Mani-Sadhasivam) [inline README.md]: What does x3 mean? 3 boards? In that case use below:

[Any 96Boards CE](https://www.96boards.org/products/ce/) - 3 Nos

**Review by Mani-Sadhasivam — CHANGES_REQUESTED** [review]
Looks good! Just a small nitpick.

**Comment by ric96** [comment]
@Mani-Sadhasivam I made the requested changes...

File: README.md [hunk 2]
```diff
@@ -1,3 +1,75 @@
...
+- [2) Software](#2-software)
+   - [2.1) Client Build Environment Setup](#21-client-build-environment-setup)
+   - [2.2) Database Server Setup](#22-api-keys)
+   - [2.3) Running the Application](#23-running-the-application)
+
+# 1) Hardware
+
+## 1.1) Hardware Requirements
+
+- [Any 96Boards CE](https://www.96boards.org/products/ce/) 3 Nos
```
Comment (Mani-Sadhasivam) [inline README.md:17]: please mention the OS here

**Review by Mani-Sadhasivam — CHANGES_REQUESTED** [review]

**Review by Mani-Sadhasivam — APPROVED** [review]

**Pull request merged**